dev-requirements.txt

```diff
@@ -1,4 +1,6 @@
 pytest~=0.6.2
 pytest-dbt-adapter==0.5.1
 dbt-core~=0.21.0
-.
\ No newline at end of file
+isort
+flake8
+black
```

Comment:
we should really pin these, to at least a minor version and letting patches float. i assume this is the file used by GH actions to install the necessary packages? we also don't need to install the adapter itself anymore with `.`?

Reply:
Pinning: I actually don't think that's super important for linting, but I'm also not going to argue. I asked for versions >= the current point versions.

If we're using `setup.py`, i.e. `pip install -e filepath` it installs the adapter, no?